account_financial_report_webkit/migrations/8.0.1.3.0/pre-migration.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    """
+    Truncate account_static_balance, this will be recalculated in the
+    post-migration
+    """
+    if not version:
+        return
+
+    # Clearing existing static balances
+    logger.info(
+        'Remove existing static balances.')
+    cr.execute("TRUNCATE TABLE account_static_balance;")
+    logger.info(
+        'Done removing existing static balances.')
+
+    logger.info("Drop constraint")
+    cr.execute("""
+        ALTER TABLE account_static_balance 
+        DROP CONSTRAINT account_static_balance_account_static_balance_unique;
+        """)
+
+    logger.info("Droped constraint")
+    logger.info(
+        "Start creating index account_move_line(account_id, journal_id, "
+        "period_id)")
+
+    # First, create combined index
+    cr.execute("""
+        CREATE index concurrently account_move_line_combined ON
```

Comment:
Can you move this to _auto_end()? See essent_reconcile_rules/models/account_move_line.py for an example.

Reply:
Drop `concurrently` (because it will be executed during deployment) from the statement. Suggest to rename to something like `account_move_line_account_journal_period_index`